Configuration/Install/InstallSteps.yaml

```diff
@@ -10,6 +10,18 @@ databaseConnect:
     type: install
     description: 'Set up database connection'
     arguments:
+        databaseDriver:
+            description: 'Database connection type'
+            option: '--database-driver'
+            type: select
+            values:
+                mysqli: 'MySQL connection'
+                pdo_sqlite: 'SQLite connection'
```

Comment:
Maybe add `TYPO3 9.5+ only` or similar here?

Reply:
There will be an error message, when this option is selected. That is the best I can do for now, without major refactoring (not offering the option, when not available)

Reply:
Added a hint now